junebug/stores.py

```diff
@@ -21,6 +22,11 @@ def _redis_op(self, func, id, *args, **kwargs):
         yield self.redis.expire(id, self.ttl)
         returnValue(val)
 
+    def get_key(self, channel_id, message_id):
+        '''Gets the key where a message would be stored, using the channel and
+        message ids'''
+        return '%s:%s:%s' % (channel_id, self.KEYSPACE, message_id)
```

Comment:
The reason I changed to putting it by the base class is that I would be repeating essentially the same thing in all the subclasses, and I wanted to reduce that repetition. Maybe the base class should have a method that takes in n strings, and returns a key, which the subclasses can use in their own key making functions.